app/assets/stylesheets/templates/_ctc-home.scss

```diff
@@ -18,10 +18,34 @@
     padding-bottom: 0;
   }
 
-  .slab--why {
+  .slab--affiliations {
     background-color: $color-grey-light;
   }
 
+  .affiliations {
+    display: flex;
+    justify-content: space-around;
```

Comment:
what happens if you do space between here instead of space around? If the grid is restricted to a max-width already, you should get the outer two to align with edges a bit better.